Dockerfile.aarch64

```diff
@@ -0,0 +1,78 @@
+# Specify the base image to support multi-arch images, such as
+# - 'quay.io/condaforge/linux-anvil-aarch64' for Linux aarch64
+# - 'quay.io/condaforge/linux-anvil-cos7-x86_64' for Linux x86_64
+ARG BASE_IMAGE=quay.io/condaforge/linux-anvil-aarch64
+
+FROM quay.io/condaforge/linux-anvil-aarch64 as base
+
+# Copy over C.UTF-8 locale from our base image to make it consistently available during build.
```

Comment:
The comment says `from our base image` but it uses totally different image - `bioconda/base-glibc-busybox-bash`.
Is it possible to use ` $BASE_IMAGE` instead ?

Reply:
This base image is not `${BASE_IMAGE}`, I'm not sure this should be address or not, this is copy from x86 dockerfile. So if we change this, we can fix them in a separate PR.